### PR TITLE
Input accent crop

### PR DESCRIPTION
**Pull request by andreapiras**
## 🖼 Context

Input is currently cropping some character OÜ -> OU
image

Fixed 
image

## 🚀 Changes

 
Changed the input from height:48px to min-height:48px and max-height:48px;

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -168,7 +168,6 @@ workflows:
                 - main
 
       - deploy-docs-for-branch:
-          type: approval
```
Comment (andreapiras) [inline .circleci/config.yml:171]: This is not working branches are not getting deployed

File: packages/css/src/less/mixins/_forms.less [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  }
+}
+
+// Form control sizing
+//
+// Relative text size, padding, and border-radii changes for form controls. For
+// horizontal sizing, wrap controls in the predefined grid classes. `<select>`
+// element gets special love because it's special, and that's a fact!
+.input-size(@input-height; @input-padding; @font-size; @line-height; @border-radius) {
+  min-height: @input-height;
```
Comment (andreapiras) [inline packages/css/src/less/mixins/_forms.less:109]: This has been changed from 
`height: @input-height;`
to 
`min-height: @input-height;
  max-height: @input-height;`

**Comment by helendurranttw** [comment]
I just tested on the deploy, doesn't look like it's fixed 😞 I think height is still being set somewhere? Also, a brief look at the code turned up another place (.select.form-control) where height is being set - make sure you also test inputs used inside the Select component.

**Review by helendurranttw — CHANGES_REQUESTED** [review]
- Take a look for the rogue height which is still being set on .form-control.
- Also test inside Select - I think it is an issue here too
- Could you update _forms.less to explain that this was copied directly from Bootstrap - and also add a comment explaining why we're using min-height / max-height instead of height.
- Lastly - have you tested across browsers? I didn't test that fix thoroughly

**Comment by andreapiras** [comment]
> I just tested on the deploy, doesn't look like it's fixed 😞 I think height is still being set somewhere? Also, a brief look at the code turned up another place (.select.form-control) where height is being set - make sure you also test inputs used inside the Select component.
Locally on my docs it works fine 
image
I can't see the css on the branch docs so I think it's a cache issue

**Comment by andreapiras** [comment]
> * Take a look for the rogue height which is still being set on .form-control.
> * Also test inside Select - I think it is an issue here too
image

> * Could you update _forms.less to explain that this was copied directly from Bootstrap - and also add a comment explaining why we're using min-height / max-height instead of height.
Not imported anymore replaced by overrides

> * Lastly - have you tested across browsers? I didn't test that fix thoroughly
Yes IE11,FF,Chrome e Safari

**Review by helendurranttw — APPROVED** [review]
Rename the mixin and you're good to go

**Pull request merged**